File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/diag/dto/didefdto/d/DidefDto.java
package iih.ci.diag.dto.didefdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class DidefDto extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId_di() {
		return ((String) getAttrVal("Id_di"));
	}
	/**
	 * 主键
	 * @param Id_di
	 */
	public void setId_di(String Id_di) {
		setAttrVal("Id_di", Id_di);
	}
	/**
	 * 名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 编码
	 * @return String
	 */
	public String getCode() {
		return ((String) getAttrVal("Code"));
	}
	/**
	 * 编码
	 * @param Code
	 */
	public void setCode(String Code) {
		setAttrVal("Code", Code);
	}
	/**
	 * displaynam4
	 * @return String
	 */
	public String getName4() {
		return ((String) getAttrVal("Name4"));
	}
	/**
	 * displaynam4
	 * @param Name4
	 */
	public void setName4(String Name4) {
		setAttrVal("Name4", Name4);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.diag/dto/didefdto/d/DidefDto.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.diag.dto.didefdto.d
{
    /// <summary>
    /// DidefDto 的摘要说明。
    /// </summary>
    public class DidefDto : BaseDTO {

        public DidefDto() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_di {
            get { return getAttrVal<string>("Id_di",null); }
            set { setAttrVal<string>("Id_di", value); }
        }

        /// <summary>
        /// 名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 编码
        /// </summary>
		public string Code {
            get { return getAttrVal<string>("Code",null); }
            set { setAttrVal<string>("Code", value); }
        }

        /// <summary>
        /// displaynam4
        /// </summary>
		public string Name4 {
            get { return getAttrVal<string>("Name4",null); }
            set { setAttrVal<string>("Name4", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_di", "Name", "Code", "Name4"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.diag.dto.didefdto.d.DidefDto";
        }
    }
}
